integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/withapim/BackendSecurityTestCase.java

```diff
@@ -122,14 +128,30 @@ public void testBasicAuthSecuredBackend() throws CCTestException, MalformedURLEx
         Assert.assertTrue(respHeaders.containsKey("authorization"), "Backend did not receive auth header");
         Assert.assertEquals(respHeaders.get("authorization"), "Basic YWRtaW46YWRtaW4=",
                 "backend basic auth header is incorrect");
+    }
 
-        //test 2 - invoke non secured resource
-        endpoint = Utils.getServiceURLHttps(API_CONTEXT + "/1.0.0/echo2");
-        response = HttpsClientRequest.doGet(endpoint);
+    @Test(description = "oauth2 secured resource backend basic auth")
```

Comment:
the same description of the previous test case (line 118)

Reply:
fixed